engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -201,4 +201,15 @@ object ProcessCompilationError {
 
   case class CannotCreateObjectError(message: String, nodeId: String) extends ProcessCompilationError with InASingleNode
 
+  case class CustomParameterValidationError(message: String, description: String, paramName: String, nodeId: String) extends ParameterValidationError with InASingleNode
```

Comment:
Can we "merge" it with CustomeNodeError above?

Reply:
Good idea, I've removed this one and reused CustomNodeError